frontend/public/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -43,7 +43,7 @@ export class OperatorHubSubscribeForm extends React.Component<OperatorHubSubscri
         : preferredInstallMode, InstallModeType.InstallModeTypeSingleNamespace);
     let targetNamespace = this.state.targetNamespace || this.props.targetNamespace;
     if (installMode === InstallModeType.InstallModeTypeAllNamespaces) {
-      targetNamespace = _.get(this.props.operatorGroup, 'data', [] as OperatorGroupKind[]).find(og => isGlobal(og)).metadata.namespace;
+      targetNamespace = 'openshift-operators';
```

Comment:
We can hard-code the `metadata.name` of the `OperatorGroup`, which is the [same upstream and downstream](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/deploy/chart/templates/0000_50_olm_13-operatorgroup-default.yaml#L4).